plugins/pfring/README.md

```diff
@@ -0,0 +1,41 @@
+# PF_RING Plugin Capture Plugin
+
+## Building
+
+To build this plugin, built Suricata with the `--enable-pfring` and
+optionally the `--with-libpfring-includes` and
+`--with-libpfring-libraries` command line options.
+
+## Running
+```
+/usr/local/suricata/bin/suricata \
+    --set plugins.0=/usr/local/lib/suricata/pfring.so \
+    --capture-plugin=pfring-plugin \
+    --set pfring.0.interface=eno1
+```
+
+### --set plugins.0=/usr/local/lib/suricata/pfring.so
+
+This command line option tells Suricata about this plugin. This could also
+be done in `suricata.yaml` with the following section:
+```
+plugins:
+  - /usr/local/lib/suricata/pfring.so
+```
+
```

Comment:
I'm thinking we should do make Suricata smarter with respect to plugins.  Provided plugins following a naming scheme, and `--capture-plugin=qwerty` is provided, and we don't know what `qwerty` is, go and look for the plugin.